ex1/ex1.c

```diff
@@ -9,6 +9,15 @@
 int main(int argc, char *argv[])
 {
     // Your code here
+    char str[100] = "What does fork() do\n";
+    int num = 100;
+    printf(str);
+    printf(num)
```

Comment:
This is correct, but VSCode recommends printf("%d", num); for security. Also, semicolon at the EOL